### ISSUE TITLE
Does this work with SPT-AKI?

### ISSUE THREAD

**Issue by GH-vanny** [sailro/EscapeFromTarkov-Trainer#146]
I was just wondering if it is known that this works with SPT-AKI version of tarkov. If so- I'm running on SPT-AKI 2.2.1 v0.12.12.15.16778. Any info?

**Comment by sailro** [comment]
Yes it is working. And I just tried with the latest SPT-AKI 2.2.3 and EFT 0.12.12.16909
